docs/features/opengraph/api/custom-post-type-filters.md

```diff
@@ -0,0 +1,119 @@
+---
+id: wpseo-opengraph-post-type-filters
+title: "Yoast SEO: Change the OpenGraph tags for post type"
+sidebar_label: Alter OpenGraph output for post type
+---
+These filters allow altering an existing OpenGraph output for a specific post type. <br />
+They are used in `Yoast SEO Premium` for Social media appearance for the default post, with priority 10. When using with `Yoast SEO Premium`, make sure to use the right priotity to avoid conflicts.
```

Comment:
```suggestion
They are used in `Yoast SEO Premium` for Social media appearance for the default post, with priority 10. When using with `Yoast SEO Premium`, make sure to use the right priority to avoid conflicts.
```